CHANGELOG.d/README.md

```diff
@@ -0,0 +1,77 @@
+# About
```

Comment:
I think this file needs to move into the governance or some other centralized repository so that it isn't duplicated in every repo using this model. Instead, this README file can just have a brief description (the paragraph below is fine), and then a link over to the centralized document.